api/data_explorer/util/facets_util.py

```diff
@@ -0,0 +1,152 @@
+from elasticsearch_dsl import Search
```

Comment:
Do you mind renaming this to elasticsearch_util.py? I'm adding elasticsearch_util.py in another PR.

Then we don't have to worry about which file to put something in. For example, technically get_field_type() is not related to facets and should be in elasticsearch_util.